config.js

```diff
@@ -94,7 +94,12 @@ config.mqtt = {
     /**
      * Time to keep connection open between client and MQTT broker (default is 0 seconds)
      */
-    keepalive: 0
+    keepalive: 0,
+
+    /**
+     * Whether to use slashes at the beginning of topic when sending or not
+     */
+    avoidLeadingSlashes: true
```

Comment:
In order to keep backward compatibility the default for this setting should be false, isn't it?

Reply:
In addition, what it would be better... avoidLeadingSlashes or avoidLeadingSlash? (because I think the character is avoided is a single '/'...)

Reply:
Absolutely right about the compatibility. I thought I got it right.

Reply:
About the naming convention is a semantic aspect. I leave it to you to decide. I chose the plural form because it refer to many topics, but I get your point that it is only one slash per topic

Reply:
I think is better singular (slash).

Reply:
Done.